Automattic/wp-calypso
# Import: Rename Engine6 to GoDaddy and site-importer to Wix

See D26547-code for corresponding server-side changes.

Note: The feature-flag was intentionally unchanged, as it'll be removed when the GoDaddy GoCentral importer launches, and anyone using the importer currently is aware of the `manage/import/engine6` feature-flag

#### Changes proposed in this Pull Request

* Renames instances of "site-importer" that refer specifically to the wix engine as such.
* Renames engine6, importer6, etc to "godaddy-gocentral"
* Adds logo for godaddy-gocentral
* Refactors how logos for site-importer importers are handled
* Removes several obsoleted `tracksType` constants that were used to add `-wix` to the end of Tracked importer ids when the type ended with `site-importer` (which, it no longer does)
* Reorders importer list such that WordPress is first, and then the remainder of the list is alphabetical

![image](https://user-images.githubusercontent.com/363749/55442051-a71f7480-5572-11e9-8e1c-50ec22a69b20.png)


#### Testing instructions

* Run branch and visit `/settings/import`
* Verify that all logos are correct.
* Verify that WordPress is listed first, followed by the other importers in alphabetical order.
* Verify that a Wix import still works as usual
* Follow the test-plan from D26547-code.
